package.json

```diff
@@ -12,7 +12,7 @@
     "prepare": "husky install"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx,vue,yaml}": "eslint --fix",
+    "*.{js,jsx,ts,tsx,vue}": "eslint --fix",
```

Comment:
Removed Yaml files from Eslint, because it doesn't need a logical error checks.
And it still remains in the prettier.